web/src/lib/components/shared-components/version-announcement-box.svelte

```diff
@@ -15,22 +16,30 @@
     return `v${major}.${minor}.${patch}`;
   }
 
-  function onAcknowledge() {
+  const onAcknowledge = () => {
     // Store server version to prevent the notification
     // from showing again.
     localStorage.setItem('appVersion', githubVersion);
     showModal = false;
-  }
+  };
 
   onMount(async () => {
     try {
-      githubVersion = await getGithubVersion();
+      const { data } = await api.userApi.getLatestImmichVersionAvailable();
+      if (data.availableVersion) {
+        githubVersion = semverToName({
+          major: data.availableVersion.major,
+          minor: data.availableVersion.minor,
+          patch: data.availableVersion.patch,
+        });
+      }
+
       if (localStorage.getItem('appVersion') === githubVersion) {
         // Updated version has already been acknowledged.
         return;
       }
 
-      if (githubVersion !== serverVersionName) {
+      if (data.availableVersion && !disable_check && !import.meta.env.DEV) {
```

Comment:
Does this work? Only environment variables that start with PUBLIC_* are available on the web afaik.

Reply:
It doesn't. I wanted to use it just to make sure, but it's useless